test/quicer_snb_SUITE.erl

```diff
@@ -66,6 +66,17 @@
 -include_lib("snabbkaffe/include/snabbkaffe.hrl").
 -include("quicer.hrl").
 
+-define(my_check_trace(BUCKET, RUN, CHECK),
+    ?check_trace(
+        BUCKET,
```

Comment:
Why is this argument called BUCKET?

Reply:
copied from https://github.com/kafka4beam/snabbkaffe/blob/master/include/test_macros.hrl#L108